source/data_analysis_SPRING2023.Rmd

```diff
@@ -753,15 +785,15 @@ nat_ap <- apoidea %>%
   left_join(naturalhistorytraits, join_by(species_nm == species))
 ```
 
-```{r}
+```{r, fig.cap = "Unieke functionele groepen apoidea voor transecten (TS) vs. pan traps (PT)."}
 venn(nat_ap, "method_cd", "optional_functional_group")
```

Comment:
```suggestion
```{r, fig.cap = "Unieke functionele groepen apoidea voor transecten (TS) vs. pan traps (PT)."}
venn(nat_ap, "method_cd")
```

Reply:
Als we dit argument weglaten gaan we weer naar unieke soorten kijken ipv unieke functionele groepen, dus waarom zou je dit doen?

Reply:
Ah juist ik herinner het me weer. Maar dan is er iets mis met de venn() functie . Het kan bij mij niet knitten. Dus wellicht heb je wijzigingen gemaakt aan de functie die niet zijn doorgekomen?

Reply:
Er staat nu "optional_functional_group", maar dat is geen kolomnaam uit de data.frame...
Je moet daar verwijzen naar bv "foraging_preference", of "sociality". Maar vermits er meerdere kolommen (functionele classificaties) mogelijk zijn, is dit iets dat je beter via een parent-child constructie oplost op dezelfde manier als ik dat eerder heb gedaan.

Reply:
Dus er is niets mis met de venn() functie, die aanpassingen waren wel doorgekomen. Er wordt enkel iets foutief doorgegeven. In een child document kan je wel verwijzen naar "{{optional_functional_group}}" door te werken met {{}}. Je leest best eens heel aandachtig de documentatie over parent child documents en hoe ze juist werken.

Reply:
Zie vooral de help file van `knit_expand()`